docs/blockchain/contracts/nft-contract/nft-actions/stofrcfg.a.experimental.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'stofrcfg.a'
+order: 36
+
+---
+
+# stofrcfg.a
+
+Set global uniq offer configuration
+
+## Behavior
+
+Set global configuration for all uniq offer include both token and token factory.
+
+## Technical Behavior
+
+Only `ultra.nft.ft` account can set buy offer config
+
+The action stores the config in to `offercfg.a` singleton with the specified arguments.
```

Comment:
Should be
The action stores the configurations to `offercfg.a` singleton table ...

Reply:
updated